flink-python/pyflink/table/table_config.py

```diff
@@ -44,18 +47,55 @@ def parallelism(self):
     def parallelism(self, parallelism):
         self._parallelism = parallelism
 
+    @property
+    def timezone(self):
+        return self._j_table_config.getTimeZone().getID()
+
+    @timezone.setter
+    def timezone(self, timezone_id):
```

Comment:
Add some Doc, such as:

`the timezone_id for a timezone, either an abbreviation such as "PST", a full name such as "America/Los_Angeles", or a custom timezone_id such as "GMT-8:00". `

What do you think?

Reply:
Done.